db_harmonisation/crude_db_harmonisation.py

```diff
@@ -9,6 +9,7 @@
 from Bio.Seq import translate, Seq
 from construct_megares_mapping import construct_megares
 from construct_groot_mappings import get_groot_aro_mapping
+import pandas as pd
```

Comment:
This is kind of slow. Can you move it to inside the functions to avoid slowing down operations like `jug status`?